### ISSUE TITLE
Text in draggables escaped

### ISSUE THREAD

**Issue by aperitero** [h5p/h5p-drag-text#1]
If you create a Drag & Drop Text with the following text:
"I'm not an human. \*I'm\* a computer."

The draggable displays "I\&#039;m" (while the first apostrophe is displayed correctly in the text).

Here is what I changed to get it working:
```
--- drag-text.original.js	2015-03-03 17:01:17.016514773 +0200
+++ drag-text.js	2015-03-03 17:14:42.300507937 +0200
@@ -451,7 +451,7 @@
 
     //Make the draggable
     var $draggable = $('<div/>', {
-      text: answer,
+      html: answer,
       'class': DRAGGABLE
     }).draggable({
       revert: function (isValidDrop) {
```

**Comment by icc** [comment]
You're correct. The content parameters that comes into H5Ps have already been stripped server side, and should be inserted as html. Can you fix thomasmars?

Thank you for pointing it out :-)

**Comment by thomasmars** [comment]
Hello. Thanks for the feedback :+1: 

This is now fixed in d0920aa987cd6a89ecc01b3f817352476ca31d2f

**Comment by icc** [comment]
I see that there are 3 more buttons that have the same issue.

**Comment by thomasmars** [comment]
Properly changed all text fields to html fields. This should do it :)

Available at 6aba390ddb6a3b71658b8dfdc40ac508ce38e07c
